graph/topo/bench_test.go

```diff
@@ -107,3 +121,71 @@ func BenchmarkUndirectedCyclesInGnp_10_half(b *testing.B) {
 func BenchmarkUndirectedCyclesInGnp_100_half(b *testing.B) {
 	benchmarkUndirectedCyclesIn(b, gnpUndirected_100_half)
 }
+
+func benchmarkSort(b *testing.B, g graph.Directed) {
+	for i := 0; i < b.N; i++ {
+		_, _ = Sort(g)
+	}
+}
+
```

Comment:
```suggestion
```

Reply:
Done!

I've also applied this change to the other collections of functions in this file that have a lowercase `benchmark*` function and a few camel-case `Benchmark*` functions, like `DirectedCyclesIn` and `TarjanSCC`.